raiden/transfer/state_change.py

```diff
@@ -231,21 +231,30 @@ def __init__(
             self,
             pseudo_random_generator,
             block_number: typing.BlockNumber,
+            chain_id: typing.ChainID,
     ):
         if not isinstance(block_number, int):
             raise ValueError('block_number must be int')
 
+        if not isinstance(chain_id, int):
+            raise ValueError('chain_id must be int')
+
         self.pseudo_random_generator = pseudo_random_generator
         self.block_number = block_number
+        self.chain_id = chain_id
 
     def __repr__(self):
-        return '<ActionInitNode block_number:{}>'.format(self.block_number)
+        return '<ActionInitNode block_number:{} chain_id:{}>'.format(
```

Comment:
Perhaps we should renamed it now, from `ActionInitNode` to `ActionInitChainNetwork` or something similar, and if you do, `NodeState` should become `ChainNetworkState` too